configure.ac

```diff
@@ -208,6 +208,31 @@ if test "$gd_with_LIBFREETYPE" = "yes"; then
   AC_DEFINE([HAVE_FT2BUILD_H], [1], [Define if you have the ft2build.h header.])
 fi
 
+dnl CTL configure tests
+AC_ARG_ENABLE(ctl,
+    [AS_HELP_STRING([--disable-ctl],
+                    [disable complex text layout support])],
+    [enable_ctl=$enableval],
+    [enable_ctl='yes'])
+
+if test "x$gd_with_LIBFREETYPE" != "xyes" -a "x$enable_ctl" != 'xno'; then
+    AC_MSG_ERROR([Complex text support requested but FreeType is missing])
+fi
+
+have_ctl=""
+if test "x$enable_ctl" != 'xno'; then
+    PKG_CHECK_MODULES(CTL, [fribidi harfbuzz], have_ctl="yes", [])
```

Comment:
split this up into two lib checks and use standard vars:
``PKG_CHECK_MODULES([FRIBIDI], [fribidi], [gd_with_FRIBIDI=yes], [gd_with_FRIBIDI=no])``

then you can do:

    if test "x$gd_with_FRIBIDI$gd_with_HARFBUZZ" != "xyesyes"; then
      if test "x$gd_enable_CTL" = "xyes"; then
        AC_MSG_ERROR(...)
      fi
      gd_enable_CTL="no"
    fi